inference-engine/thirdparty/clDNN/tests/test_cases/fusings_gpu_test.cpp

```diff
@@ -3579,3 +3579,115 @@ INSTANTIATE_TEST_CASE_P(fusings_gpu, deconv_scale_actv_quant_u8_eltw_scale_actv_
+#define CASE_ELTWISE_I8_2   {2, 16, 4, 4}, data_types::i8,  format::bfzyx,          data_types::f32,  format::bfzyx
+#define CASE_ELTWISE_U8_1   {2, 16, 4, 4}, data_types::u8,  format::bfyx,           data_types::f32,  format::bfyx
+#define CASE_ELTWISE_U8_2   {2, 16, 4, 4}, data_types::u8,  format::bfzyx,          data_types::f32,  format::bfzyx
+
+#define eltwise_tests(case) \
+    eltwise_test_params{case, eltwise_mode::sum, 2, 3}, \
+    eltwise_test_params{case, eltwise_mode::sub, 2, 3}, \
+    eltwise_test_params{case, eltwise_mode::prod, 2, 3}, \
+    eltwise_test_params{case, eltwise_mode::div, 2, 3}
+
+// not supported for integer types.
+#define eltwise_tests_fp32(case) \
+    eltwise_test_params{case, eltwise_mode::pow, 2, 3}, \
+    eltwise_test_params{case, eltwise_mode::squared_diff, 2, 3}, \
+    eltwise_test_params{case, eltwise_mode::max, 2, 3}, \
+    eltwise_test_params{case, eltwise_mode::min, 2, 3}, \
+    eltwise_test_params{case, eltwise_mode::mod, 2, 3}, \
+    eltwise_test_params{case, eltwise_mode::floor_mod, 2, 3}
+
+class EltwiseFusingTest : public ::BaseFusingTest<eltwise_test_params> {
+public:
+    void execute(eltwise_test_params& p) {
+        auto input_prim = get_mem(get_input_layout(p));
+
+        network network_not_fused(this->engine, this->topology_non_fused, bo_not_fused);
+        network network_fused(this->engine, this->topology_fused, bo_fused);
+
+        network_fused.set_input_data("input", input_prim);
+        network_not_fused.set_input_data("input", input_prim);
+
+        compare(network_not_fused, network_fused, p);
+    }
+
+    layout get_input_layout(eltwise_test_params& p) { return layout{p.input_type, p.input_format, p.input_size}; }
+
+    layout get_per_channel_layout(eltwise_test_params& p) {
+        return layout{p.default_type, p.default_format, tensor{1, p.input_size.feature[0], 1, 1}};
+    }
+};
+
+class eltwise_quantize : public EltwiseFusingTest {};
+TEST_P(eltwise_quantize, basic) {
+    auto p = GetParam();
+    create_topologies(input_layout("input", get_input_layout(p)),
+                      data("input2", get_mem(get_input_layout(p))),
+                      eltwise("eltwise", {"input", "input2"}, p.mode),
+                      data("in_lo", get_mem(get_single_element_layout(p), min_random, 0)),
+                      data("in_hi", get_mem(get_single_element_layout(p), 1, max_random)),
+                      data("out_lo", get_mem(get_single_element_layout(p), 0)),
+                      data("out_hi", get_mem(get_single_element_layout(p), 255)),
+                      quantize("quantize", "eltwise", "in_lo", "in_hi", "out_lo", "out_hi", 256, data_types::u8),
+                      reorder("out", "quantize", p.default_format, data_types::f32));
+
+    tolerance = 1.f;
+    execute(p);
+}
+
+class eltwise_quantize_int8 : public EltwiseFusingTest {};
+TEST_P(eltwise_quantize_int8, basic) {
```

Comment:
1. It is also useful to test quantization with per channel "in_lo" and "in_hi".
2. Also wouldn't it be more clear to have single test case `eltwise_quantize` with two tests u8 and to test per channel range i8_per_channel?